CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# CMake version configuration
+cmake_minimum_required(VERSION 3.5...3.19)
+if(${CMAKE_VERSION} VERSION_LESS 3.12)
+    cmake_policy(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION})
+endif()
+
+# Define project
+if(${CMAKE_VERSION} VERSION_LESS 3.12)
```

Comment:
Why this if/else, can't we simply always omit `DESCRIPTION` and `HOMEPAGE_URL`?

Reply:
It could be useful for newer CMake versions, but we could also remove it if you prefer that.

Reply:
Okay so if it has no clear advantage, honestly I'd prefer to remove it to keep the cmake config simple (avoiding redundancy) :wink:

Reply:
Done!